openwisp_ipam/templates/admin/openwisp-ipam/subnet/change_list.html

```diff
@@ -9,5 +9,12 @@
   </a>
 </li>
 {% endif %}
+{% if not is_popup and has_add_permission and has_change_permission %}
+<li>
+  <a href="{% url opts|admin_urlname:'recoverlist' %}" class="recoverlink">
+    {% blocktrans with cl.opts.verbose_name_plural|escape as name %}Recover deleted {{name}}{% endblocktrans %}
+  </a>
+</li>
```

Comment:
I wonder if we can avoid having to add this here.
Can you try moving `reversion` in INSTALLED_APPS before `openwisp_ipam` and remove this code to see if it's shown without it? It may be necessary to put it before the django admin too.

Reply:
@nemesisdesign I tried by commenting out this portion of code and moving the `reversion` on top of `openwisp_ipam` in the INSTALLED_APPS section but then the **recover** button disappears.

Reply:
One way to remove this would be to change the base template `{% extends "reversion/change_list.html" %}` instead of `{% extends "admin/change_list.html" %}`

I think that should help.

Reply:
Thanx @atb00ker, it worked. :+1: